ml4h/tensormap/ukb/mri.py

```diff
@@ -5,10 +5,10 @@
 import h5py
 import numpy as np
 from tensorflow.keras.utils import to_categorical
-import cv2
-import blosc
+#import cv2
+#import blosc
```

Comment:
These two modules seem to be called by a few adiposity TMAPs in `mri.py`. Maybe we could add them as requirements instead of commenting out. No need to address this in this PR.

Reply:
Oops good catch! I uncommented and added a TODO.